engine/gamesys/src/gamesys/resources/res_textureset.cpp

```diff
@@ -56,9 +56,8 @@ namespace dmGameSystem
         }
 
         tile_set->m_TextureSet = texture_set_ddf;
-        /////////! FIXME: need obtain GraphicsContext somehow. In ResourceCreationParams registered PhysicsContext
-        uint16_t width = dmGraphics::GetOriginalTextureWidth(0, texture_res->m_Texture);
-        uint16_t height = dmGraphics::GetOriginalTextureHeight(0, texture_res->m_Texture);
+        uint16_t width = texture_res->m_OriginalWidth;
+        uint16_t height = texture_res->m_OriginalHeight;
```

Comment:
The bug fix: When I updated the PS5 backend, I got a null crash, as I used the passed graphics context.